src/scripts/run-parallel-tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [ -n "" ]; then
+    set -- "$@" --settings "$PARAM_SETTINGS_FILE"
```

Comment:
As shared by @EricRibeiro. It is best practice to wrap the `set --"$@"` pattern in set -x, so that the settings which are executed, are printed to the output.

```suggestion
    set -x
    set -- "$@" --settings "$PARAM_SETTINGS_FILE"
    set +x
```